src/resources/js/frontend-ticket-form.js

```diff
@@ -287,4 +287,14 @@ var tribe_tickets_ticket_form = {};
 	$( function() {
 		my.init();
 	} );
+
+	// Listen for any clicks on an element in the document with the `link` class
+	$( window ).on( 'click', '.button-events-list', function( e ) {
```

Comment:
`$( document )` is probably a better choice here. `document` is a descendent of `window` and keeps the targeting only slightly narrower (at the `<html>` document level).